### ISSUE TITLE
Overriding on directory - only picking rules (or at least option to)

### ISSUE THREAD

**Issue by alteregoart** [loeffel-io/ls-lint#329]
# Feature request 

Currently defining rules for a directory override all rules for the current directory and all their subdirectories. 
In my usage, I would like to inherite from previous rules defined on a previous level.

## Why
- Lot of duplicate in my configuration because of configuration of subdirectory
- This seems as a more natural approch (at least IMO)
example : 
```yaml
ls:
  .ts: camelCase
  .svg: snake_case

  models:
    .ts: camelCase
    # here svg are not seen by ls-lint because previous level rules does not apply
```

for a directory: 
```
.
├── globalSetup.ts
├── models
│  ├── NOT_RIGHT.svg
│  └── request.utility.ts
└── testEnvironment.ts
```
No error

## Expect behavior
```yaml
ls:
  .js: kebab-case
  .svg: snake_case

  models:
    .js: PascalCase
```
.svg should raise an error on directory `models` are not snake_case

**Comment by loeffel-io** [comment]
duplicate of #278
